### PR TITLE
[general] Fix build breakage on FRDM-K64F and QEMU

### LINKED ISSUE #1291: [ws] Failed to build websocket for K64F and QEMU
### Description

Building websocket samples([WebSocketServer.js](https://github.com/01org/zephyr.js/blob/master/samples/websockets/WebSocketServer.js)) for K64F and QEMU failed.

### Test Code

[WebSocketServer.js](https://github.com/01org/zephyr.js/blob/master/samples/websockets/WebSocketServer.js);

### Steps to Reproduction

1. `make BOARD=frdm_k64f JS=samples/websockets/WebSocketServer.js`
2. `make BOARD=qemu_x86 qemu JS=samples/websockets/WebSocketServer.js`

### Actual Result

selection_069

 ### Expected Result

Building websocket for k64f and qemu successfully.

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | 15d3847 | FRDM-K64F | Jun 21, 2017 | Fail
master | 15d3847 | QEMU | Jun 21, 2017 | Fail

### PR DESCRIPTION
**Pull request by jimmy-huang**
Fixes #1291

Signed-off-by: Jimmy Huang <jimmy.huang@intel.com>

### CODE REVIEW

File: src/zjs_ocf_server.c [hunk 1]
```diff
@@ -461,7 +461,7 @@ static ZJS_DECL_FUNC(ocf_register)
 
     for (i = 0; i < resource->num_types; ++i) {
         ZVAL type_val = jerry_get_property_by_index(res_type_array, i);
-        u32_t size = OCF_MAX_RES_TYPE_LEN;
+        jerry_value_t size = OCF_MAX_RES_TYPE_LEN;
```
Comment (grgustaf) [inline src/zjs_ocf_server.c]: You need jerry_create_number on this and the next two...

File: src/zjs_ocf_common.c [hunk 2]
```diff
@@ -223,7 +223,7 @@ void zjs_set_uuid(char *uuid)
 
 static void platform_init(void *data)
 {
-    u32_t size;
+    jerry_value_t size;
```
Comment (grgustaf) [inline src/zjs_ocf_common.c]: This is wrong too, you can't later assign an int to size like that. jerry_value_t is not a number, it's encoded - the bottom bits specify type.

**Comment by jimmy-huang** [comment]
supposed to be jerry_size_t not jerry_value_t

**Comment by grgustaf** [comment]
Not sure why that didn't occur to me, but glad I caught it. :D

**Pull request merged**